.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Not sure if these issue templates are necessary or we want to have some other types of issues. Just copy/pasted. I can also remove them again.